crates/proto/src/domain/note.rs

```diff
@@ -202,8 +202,10 @@ impl NetworkNote {
     }
 
     pub fn account_prefix(&self) -> NetworkAccountPrefix {
-        // SAFETY: This must succeed because this is a network note.
-        self.metadata().tag().try_into().unwrap()
+        self.metadata()
+            .tag()
+            .try_into()
+            .expect("NetworkNotes have network account prefix")
     }
```

Comment:
This method should probably go a way since now we have an explicit type for single-target network notes.